lib/prefab/logger_client.rb

```diff
@@ -47,15 +57,14 @@ def log_internal(message, path, progname, severity, log_context={}, &block)
                INTERNAL_PREFIX
              end
       begin
-        log(message, path, progname, severity, log_context, &block)
+        log(message, path, nil, severity, log_context, &block)
```

Comment:
why `progname` -> `nil` here?

Reply:
I took it out of the method sig for our internal bc we aren't using it, so we don't have one. This method sig is a bit tricky because we want one with path, but we create the SSE logger and send it to event-source to capture it's logs and that one needs to quack like a normal logger.